README.md

```diff
@@ -132,6 +132,7 @@ Instructions for implementing each feature follow.
     argparser.add_argument('-p', '--piglatin', action='store_true', help='atinLay')
     argparser.add_argument('-s', '--shout', action='store_true', help='N00B EMULATION MODE')
     argparser.add_argument('words', nargs='+', help='words to say')
+
```

Comment:
I guess I could see an argument for having it either way, but this way might improve readability.